src/many-ledger/src/storage.rs

```diff
@@ -105,8 +100,30 @@ fn _execute_multisig_tx(
             minicbor::to_vec(EmptyReturn)
         }
 
-        AccountMultisigTransaction::AccountAddFeatures(_args) => {
-            return Err(ManyError::unknown("Unsupported method."));
+        AccountMultisigTransaction::AccountAddFeatures(args) => {
```

Comment:
nit: we should DRY this at some point. It's almost the same code as in the LedgerModuleImpl.

The same comment applies to other non-multisig arms.

Reply:
Yes, the proper DRY is to move all the multisig code to the module and have storage be a dumb storage. This requires more refactoring than necessary for now.